internal/httpserver/api/schema_handlers.go

```diff
@@ -139,6 +140,40 @@ func HandleGetTelemetrySchema(schemaRepo repository.TelemetrySchemaRepository) h
 	}
 }
 
+func HandleWeaverSchemaExport(schemaRepo repository.TelemetrySchemaRepository) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		ctx := r.Context()
+		schemaId := chi.URLParam(r, "schemaId")
+
+		schema, err := schemaRepo.GetTelemetrySchema(ctx, schemaId)
+		if err != nil {
+			http.Error(w, "failed to get schema", http.StatusInternalServerError)
+			return
+		}
+
+		if schema == nil {
+			http.Error(w, "schema not found", http.StatusNotFound)
+			return
+		}
+
+		telemetry, err := schemaRepo.GetTelemetry(ctx, schema.SchemaId)
+		if err != nil {
+			http.Error(w, "failed to get telemetry", http.StatusInternalServerError)
+			return
+		}
+
+		yaml, err := weaver.GenerateYAML(telemetry, schema)
+		if err != nil {
+			http.Error(w, "failed to generate YAML", http.StatusInternalServerError)
+			return
+		}
```

Comment:
With the latest commit, now I'm getting an error generating the YAML. The error occurs because schemaRepo.GetTelemetry returns nil telemetry.

I don't know if I'm doing something wrong in my curl command or if the code is broken. The command I'm using is:

```
curl -X GET localhost:8080/api/v1/telemetries/scrape_samples_scraped/schemas/9139acbe44b11d43/weaver-schema
```

Where `scrape_samples_scraped` is the name of the metric I'm looking for and `9139acbe44b11d43` is the schemaID I found in the catalog UI.

<img width="621" height="815" alt="image" src="https://github.com/user-attachments/assets/52196b6c-69e0-449a-b0f6-caaab723154c" />